package.json

```diff
@@ -61,6 +61,7 @@
     "libchanged": "test -z \"$(git diff HEAD)\" || (git diff HEAD && false)",
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
+    "prebuild": "prettier --write \"{bin,src,test,example}/**/*.js\" *.js",
```

Comment:
Prettier and eslint are integrated using the prettier plugin:
https://github.com/yarax/swagger-to-graphql/blob/49a8ad869701b491cfa8abb00190edf16b378375/.eslintrc.js#L3

This means that `npm run lint:fix` also fixes prettier issues.

Also I prefer keeping linting a separate run tasks from the build.

Reply:
Oh, you're right, I missed that. I'll close this.